browser/src/control/Control.Zotero.js

```diff
@@ -78,6 +94,37 @@ L.Control.Zotero = L.Control.extend({
 		return Object.keys(this.citations);
 	},
 
+	getCitationKeysForBib: function() {
+		var uncitedKeys = [];
+		this.settings.bib.uncited.forEach(function(item) {
+			uncitedKeys.push(item[0].substring(item[0].lastIndexOf('/')+1));
```

Comment:
it would be good to create helper function for getting key  (it's repeated few times)